### ISSUE TITLE
Unable to get it working with django 4.2

### ISSUE THREAD

**Issue by samucancld** [django-json-api/django-rest-framework-json-api#1144]
## I'm trying to install the version 6.0.0 with django 4.2 but getting dependency problem

```
#0 5.267 ERROR: Cannot install -r /requirements.txt (line 11), -r /requirements.txt (line 14), -r /requirements.txt (line 2), -r /requirements.txt (line 4) and Django<4.3.0 and >=4.2.0 because these package versions have conflicting dependencies.
#0 5.267
#0 5.267 The conflict is caused by:
#0 5.267     The user requested Django<4.3.0 and >=4.2.0
#0 5.267     djangorestframework 3.14.0 depends on django>=3.0
#0 5.267     django-filter 23.1 depends on Django>=3.2
#0 5.267     django-stubs 1.16.0 depends on django
#0 5.267     djangorestframework-jsonapi 6.0.0 depends on django<4.2 and >=3.2
```

This is my requirements.txt file:
```
Django>=4.2.0,<4.3.0
djangorestframework>=3.14.0,<3.15.0
markdown>=3.4.3,<3.5.0
django-filter>=23.1.0,<23.2.0
psycopg[binary]>=3.1.8,<3.2.0

flake8>=6.0.0,<6.1.0
black>=23.3.0,<23.4.0
isort>=5.12.0,<5.13.0
mypy>=1.2.0,<1.3.0
django-stubs>=1.16.0,<1.17.0
djangorestframework-stubs>=1.10.0,<1.11.0

djangorestframework-jsonapi>=6.0.0,<6.1.0
```
Amazing library btw

**Comment by xulww** [comment]
The current version of DRF JSON API doesn't support Django 4.2. Already there's a merged [commit](https://github.com/django-json-api/django-rest-framework-json-api/commit/1182281750e92777e37fd3701d35085943125b24) in the main branch that fixes this. We will have to wait for the next release. It's just a matter of time...

**Comment by sliverc** [comment]
Currently, Django REST framework has not released a new version yet supporting Django 4.2 so we currently cannot release either.

See our discussion at https://github.com/django-json-api/django-rest-framework-json-api/pull/1142 and the open issue https://github.com/encode/django-rest-framework/pull/8794

**Comment by sliverc** [comment]
Yep let's close it. Thanks.
